model-deployment/containers/nim/README.md

```diff
@@ -54,10 +58,10 @@ When experimenting with new frameworks and models, it is highly advisable to att
 
 ## Deploy on OCI Data Science Model Deployment
 
-Once you built and pushed the NIM container, you can now use the `Bring Your Own Container` Deployment in OCI Data Science to deploy the Llama3 model.
+Once NIM container is pushed, you can now use the `Bring Your Own Container` Deployment in OCI Data Science to deploy the Llama3 model.
```

Comment:
here we are building and pushing
my comment for removing build was only other Readme